bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-1/Wprime_Wh_WhadhVV2l2nu/Wprime_Wh_WhadhVV2l2nu_narrow_M/Wprime_Wh_WhadhVV2l2nu_narrow_M_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model Vector_Triplet_free_Width_UFO
+define q = u c d s b u~ c~ d~ s~ b~
+define vc = vc+ vc-
+define vw = w+ w-
+define lep+ = e+ mu+ ta+
+define lep- = e- mu- ta-
+generate p p > vc, (vc > h vw, vw > q q, (h > z > lep+ lep- vl vl~ / vz vc ta+ ta-))
+add process p p > vc, (vc > h vw, z > q q, (h > vw > lep+ lep- vl vl~ / vz vc ta+ ta-))
```

Comment:
@agrohsje Sorry for the late review.
@clelange Can you double check this part? there's no Z in this process but you decay Z to q q. Perhaps a typo.

Reply:
Good catch, I've fixed it!